sender/sender.cpp

```diff
@@ -0,0 +1,23 @@
+
+#include <iostream>
+#include "../sensor/temperature.h"
+#include "../sensor/state_of_charge.h"
+#include <vector>
+#include "sender.h"
+
+using namespace std;
+
+bool sendSensorDataToConsole(){
+  bool isDataSendSuccessfully = false;
+  vector<int> temperatureValues = generateTemperatureValues(TEMPERATURE_MAX_VALUE ,TEMPERATURE_MIN_VALUE, READINGS_IN_STREAM);
+  vector<int> socValues = generateSocValues(SOC_MAX_VALUE ,SOC_MIN_VALUE, READINGS_IN_STREAM);
+
+  if((temperatureValues.size() == READINGS_IN_STREAM) && (socValues.size() == READINGS_IN_STREAM)) {
+    cout<<"Temperature, SOC"<<endl;
```

Comment:
WOuld be better if SOC can be expanded to STATE OF CHARGE, so that it could more readable in the Console Output.

Reply:
changes done.